Web/Presenters/GroupPresenter.php

```diff
@@ -170,6 +170,11 @@ function renderEdit(int $id): void
             $club->setShortcode(empty($this->postParam("shortcode")) ? NULL : $this->postParam("shortcode"));
 	        $club->setWall(empty($this->postParam("wall")) ? 0 : 1);
             $club->setAdministrators_List_Display(empty($this->postParam("administrators_list_display")) ? 0 : $this->postParam("administrators_list_display"));
+            if(strpos($this->postParam("website") ?? "", "https://") === 0 || strpos($this->postParam("website") ?? "", "http://") === 0) {
```

Comment:
Use `!` before func instead of `=== 0` here

Reply:
Here we need a strict comparison, since this method returns false if it does not find a string at the beginning

Reply:
oh k